src/sdk.ts

```diff
@@ -141,31 +134,19 @@ export class DefaultRelayingServices implements RelayingServices {
         try {
             const acceptToken =
                 smartWalletDeployVerifier.methods.acceptToken(tokenAddress);
-            log.log(acceptToken);
+            log.info(acceptToken);
             await acceptToken.send({ from: account });
-        } catch (error: any) {
-            log.log('Error');
-            log.log(error);
-            const reason = await getRevertReason(error.receipt.transactionHash);
-            log.error(
-                'Error adding token with address ' +
-                    tokenAddress +
-                    ' to allowed tokens on smart wallet deploy verifier',
-                reason
-            );
-            throw error;
-        }
-        try {
+
             await smartWalletRelayVerifier.methods
                 .acceptToken(tokenAddress)
                 .send({ from: account });
         } catch (error: any) {
-            log.log(error);
+            log.error(error);
             const reason = await getRevertReason(error.receipt.transactionHash);
             log.error(
                 'Error adding token with address ' +
                     tokenAddress +
-                    ' to allowed tokens on smart wallet relay verifier',
+                    ' to allowed tokens on smart wallet deploy verifier',
```

Comment:
is there a way to know which verifier threw the error? Is there a property in 'reason' that says so, perhaps?

Reply:
Not sure, you may check it otherwise, we may find some workaround (like setting a bool variable just after the first attempt) to avoid the code duplication.

Reply:
this is already done btw